### PR TITLE
add MS Outlook (Office 365) docs

### PR DESCRIPTION
**Pull request by SgtMarmite**
I don't think Proofreading is necessary here, since it is just copypasta from an already existing documentation.

What we should maybe discuss is the password field that is not used with oAuth.

### CODE REVIEW

File: components/extractors/communication/index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ The following extractors support communication systems:
 - [Gmail Messages](/components/extractors/communication/gmail/) 
 - [Google Calendar](/components/extractors/communication/google-calendar)
 - [Intercom](/components/extractors/communication/intercom/) 
+- [MS Outlook (Ofiice 365)](/components/extractors/communication/ms-outlook/)
```
Comment (davidesner) [inline components/extractors/communication/index.md:16]: @SgtMarmite prosim fix typo [MS Outlook (**Ofiice** 365)]

File: components/extractors/communication/ms-outlook/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+title: Microsoft Outlook (Office 365)
+permalink: /components/extractors/communication/ms-outlook/
+redirect_from:
+    - /extractors/communication/ms-outlook/
+---
+
+* TOC
+{:toc}
+
+Microsoft Outlook extractor for Office 365
```
Comment (davidesner) [inline components/extractors/communication/ms-outlook/index.md:11]: @SgtMarmite nejsou tam nejake prerequisites? Je neco potreba povolit na strane Outlook? (e.g. IMAP) Stalo by za to zminit a v uvodu rict, ze je to zalozeny na IMAP

**Review by davidesner — CHANGES_REQUESTED** [review]
lgtm, just few minor comments. I cannot modify the PR

File: components/extractors/communication/index.md (hunk 1 above)
Comment (SgtMarmite) [reply components/extractors/communication/index.md]: opraveno

**Comment by SgtMarmite** [comment]
> lgtm, just few minor comments. I cannot modify the PR

I have pushed changes to by branch - _SgtMarmite/connection-docs/tree/add-ex-ms-outlook-email-content-docs_ but it did not get transfered to here. Do you know if I have to create a new PR?

**Review by davidesner — APPROVED** [review]
Awesome, thanks!

**Pull request merged**